instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/Instrumenter.java

```diff
@@ -181,10 +178,6 @@ private Context doStart(Context parentContext, REQUEST request, @Nullable Instan
 
     Context context = parentContext;
 
-    spanBuilder.setAllAttributes(attributes);
-    Span span = spanBuilder.startSpan();
-    context = context.with(span);
-
     for (ContextCustomizer<? super REQUEST> contextCustomizer : contextCustomizers) {
       context = contextCustomizer.onStart(context, request, attributes);
```

Comment:
You should rename the first param of `ContextCustomizer#onStart()` to `parentContext` too